policy/datasets.yaml

```diff
@@ -935,6 +935,12 @@ fgcmStarIndices:
     tables: raw
     python: lsst.afw.table.BaseCatalog
     template: fgcm-process/fgcmStarIndices.fits
+fgcmReferenceStars:
```

Comment:
Useful info in the docstring is also which task uses it or which task makes it.

Reply:
I have updated all the fgcm descriptions with this info.